frontend/public/components/edit-yaml.jsx

```diff
@@ -29,6 +33,7 @@ import { isYAMLTemplate, getImpersonate } from '@console/dynamic-plugin-sdk';
 import { useResolvedExtensions } from '@console/dynamic-plugin-sdk/src/api/useResolvedExtensions';
 import { connectToFlags } from '../reducers/connectToFlags';
 import { errorModal, managedResourceSaveModal } from './modals';
+import ModalCodeReplace from './modals/replace-code-modal';
```

Comment:
Component name should be same as the file name. Also should be consistent with the state variable names.
```suggestion
import ReplaceCodeModal from './modals/replace-code-modal';
```